plugins/web_events/__init__.py

```diff
@@ -0,0 +1,70 @@
+from pokemongo_bot.event_manager import manager
+
+# pylint: disable=unused-variable, unused-argument
+
+bot_contexts = dict()
+enabled_events = list()
+
+
+@manager.on('bot_initialized')
+def bot_initialized_event(bot):
+    bot_contexts[bot.config.username] = bot
+
+    if isinstance(bot.config.web_events, bool):
+        if bot.config.web_events:
+            enabled_events.append('release_pokemon')
+            enabled_events.append('evolve_pokemon')
+
+
+def log(text, color='black'):
+    manager.fire('logging', text=text, color=color, prefix='Transfer')
+
+
+@manager.on('release_pokemon')
+def release(payload):
+    if 'release_pokemon' in enabled_events:
+        if 'user_id' not in payload or 'pokemon_id' not in payload and payload['user_id'] not in bot_contexts:
+            return
+
+        bot = bot_contexts[payload['user_id']]
+        inventory = bot.update_player_and_inventory()
+
+        for pokemon in inventory['pokemon']:
+            if pokemon.unique_id == payload['pokemon_id']:
+                pokemon_name = bot.pokemon_list[pokemon.pokemon_id - 1]["Name"]
+                log('[#] Releasing pokemon with id {}'.format(pokemon_name), 'green')
+
+                bot.api_wrapper.release_pokemon(pokemon_id=int(payload['pokemon_id']))
+                bot.api_wrapper.call()
```

Comment:
Instead of calling the API directly, this should just fire `transfer_pokemon` with parameter `transfer_list=[Pokemon to transfer]`.